tools/log-collector/log_collector.go

```diff
@@ -217,7 +201,8 @@ func (l *LogCollector) getResourceObjects(resourcePath string, resource *apiv1.A
 			listPath := fmt.Sprintf("%s/namespaces/%s/%s", resourcePath, l.Namespaces[index], resource.Name)
 			err := l.disClient.RESTClient().Get().AbsPath(listPath).Do(context.TODO()).Into(&obj)
 			if err != nil {
-				if errors.IsNotFound(err) || errors.IsForbidden(err) {
+				if apierrors.IsNotFound(err) || apierrors.IsForbidden(err) {
+					log.Errorf("%s", err.Error())
```

Comment:
log this on `warning` level.  Wherever `err` is ignored but needs to be logged then log it at Info or `warning` level.

Reply:
resolved